PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetHUtils.cxx

```diff
@@ -244,7 +284,8 @@ void AliAnalysisTaskEmcalJetHUtils::ConfigureEventCuts(AliEventCuts & eventCuts,
     typedef void (AliEventCuts::*MFP)();
     std::map<std::string, MFP> eventCutsPeriods = { std::make_pair("LHC11h", &AliEventCuts::SetupRun1PbPb),
                             std::make_pair("LHC15o", &AliEventCuts::SetupLHC15o),
-                            std::make_pair("LHC18qr", &AliEventCuts::SetupPbPb2018) };
+                            std::make_pair("LHC18qr", &AliEventCuts::SetupPbPb2018),
+                            std::make_pair("LHC18q", &AliEventCuts::SetupPbPb2018) };
```

Comment:
If you need LHC18q separately, I would imagine that you would want LHC18r separately here (I'm not sure you would need it separately, but it's also fine)

Reply:
Added in flag for LHC18r